game/system/newpad.cpp

```diff
@@ -43,6 +47,51 @@ u64 input_mode_key = -1;
 u64 input_mode_mod = 0;
 u64 input_mode_index = 0;
 MappingInfo g_input_mode_mapping;
+float g_frame_rate =
+    60.0f;  // frame rate reference so mouse sensitivity can be consistent on frame rate changes
+
+void SetFrameRate(float frame_rate) {
+  const float minimum_frame_rate = 0.0001f;  // Arbituary value
+  if (frame_rate < minimum_frame_rate) {
+    g_frame_rate = minimum_frame_rate;
+  } else {
+    g_frame_rate = frame_rate;
+  }
+}
```

Comment:
You should just use `Gfx::g_global_settings.target_fps` directly instead of having to manage a copy and a new function that just sets it.